build_projects/dotnet-cli-build/CompileTargets.cs

```diff
@@ -270,9 +274,30 @@ private static void RemovePdbsFromDir(string directory)
                 GenerateNuGetPackagesArchive(c, dotnet, sdkOutputDirectory);
             }
 
+            CopyMSBuildTargetsToSDKRoot(sdkOutputDirectory);
+
             return c.Success();
         }
 
+        private static void CopyMSBuildTargetsToSDKRoot(string sdkOutputDirectory)
+        {
+            var msbuildTargetsDirectory = Path.Combine(sdkOutputDirectory, "runtimes", "any", "native");
```

Comment:
"native"?  That seems odd since these aren't really native assets are they?

Also, why are they going into "runtimes\any", since these things aren't really "runtimes" assets.  It seems like they are more like normal content.